.github/workflows/release.yml

```diff
@@ -0,0 +1,97 @@
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+
+name: Release
+
+jobs:
+  initial_release:
+    name: Create base release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          draft: false
+          prerelease: false
+
+  release_assets:
+    name: Release assets
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      matrix:
+        config:
+          # Linux
+          - {os: ubuntu-latest, target: arm-unknown-linux-gnueabi, disable_tests: 1}
+          - {os: ubuntu-latest, target: armv7-unknown-linux-gnueabihf, disable_tests: 1}
+          - {os: ubuntu-latest, target: mips64el-unknown-linux-gnuabi64, disable_tests: 1}
+          - {os: ubuntu-latest, target: powerpc64le-unknown-linux-gnu, disable_tests: 1}
+          - {os: ubuntu-latest, target: x86_64-unknown-linux-musl, disable_tests: 0}
```

Comment:
don't think we need a musl version

Reply:
It was also in the travis script: https://github.com/input-output-hk/jormungandr/blob/461fad847a30eb280d887b55ba790d45fa2d181f/.travis.yml#L27